grakn-kb/src/main/java/ai/grakn/factory/GraknSessionImpl.java

```diff
@@ -115,11 +123,27 @@ public CommitLogHandler commitLogHandler(){
         return commitLogHandler;
     }
 
+    /**
+     * This methods creates a {@link GraknSessionImpl} object for the remote API.
+     * A user should not call this method directly.
+     * See {@link Grakn#session(String, String)} for creating a {@link GraknSession} for the remote API
+     * @param keyspace
```

Comment:
If you not going to describe the parameter (and in this case I don't think you should) then I would say no need to list it here. It just makes the code file longer with no gain.

Reply:
that is true. fixed in https://github.com/graknlabs/grakn/pull/2635/commits/4ae0304dcad296ebbdefd60fd19bacdec0b5b4e5